src/pages/references/changelog.md

```diff
@@ -21,6 +21,13 @@ contributors:
 
 # Changelog
 
+## 2024-10-08
+
+- Removes the experimental warnings for [`addAnimatedImage()`](./addonsdk/app-document.md#addanimatedimage) since it is now stable.
+- Adds the `author` property to [`MediaAttributes`](../references/addonsdk/app-document.md#mediaattributes) to allow it to be used with the `addAudio()` API.
+
+### New
+
```

Comment:
The New title should go above the paragraph

Reply:
Fixed!